governance/pyth_staking_sdk/src/staking/accounts.ts

```diff
@@ -0,0 +1,59 @@
+import { BorshCoder } from "@coral-xyz/anchor";
+import { PublicKey } from "@solana/web3.js";
+import type { Position, PositionAnchor } from "../types";
+import type { Staking } from "../../types/staking";
+import { POSITION_BUFFER_SIZE, POSITIONS_ACCOUNT_SIZE } from "../constants";
+import { convertBNToBigInt } from "../utils/bn";
+
+export type StakeAccountPositions = {
+  address: PublicKey;
+  data: {
+    owner: PublicKey;
+    positions: (Position | null)[];
```

Comment:
It's because of the deserialization, it can be null on chain so have to accommodate in the type. However we can then filter the null and convert to a new type that doesn't have nulls. But I think it's a bit overkill.

Reply:
Agree maybe we should filter out the nulls after deserializing

Reply:
fixed.